test/jetstream_test.go

```diff
@@ -4365,6 +4365,8 @@ func TestJetStreamSnapshotsAPI(t *testing.T) {
 	// Now connect through a cluster server and make sure we can get things to work this way as well.
 	nc2 := clientConnectToServer(t, ls)
 	defer nc2.Close()
+	// Wait a bit for interest to propagate.
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Not sure which interest you are looking for to propagate here, but keep in mind that we have helpers (maybe not in /test package?) to check/wait for interest on a literal on a server for a given account.

Reply:
ok will look into that. Thanks..